lib/enhance-assert.js

```diff
@@ -18,7 +18,8 @@ module.exports.PATTERNS = [
 	't.ok(value, [message])',
 	't.notOk(value, [message])',
 	't.same(value, expected, [message])',
-	't.notSame(value, expected, [message])'
+	't.notSame(value, expected, [message])',
+	't.snapshot(contents, [message])'
```

Comment:
Should be added to the below `NON_ENHANCED_PATTERNS` as this enhancement replaces the output, which we don't want here.